source/state_representation/test/tests/test_ellipsoid.cpp

```diff
@@ -6,16 +6,15 @@ using namespace state_representation;
 TEST(EllipsoidTest, EmptyConstructor) {
   Ellipsoid ellipse;
   EXPECT_TRUE(ellipse.get_name().empty());
-  EXPECT_TRUE(ellipse.get_center_state().is_empty());
-  EXPECT_THROW(ellipse.set_data(Eigen::Vector3d::Zero()), exceptions::EmptyStateException);
+  EXPECT_TRUE(ellipse.is_empty());
+  EXPECT_THROW(ellipse.set_data(Eigen::Vector3d::Zero()), exceptions::IncompatibleSizeException);
 
   ellipse.set_center_state(CartesianState::Identity("A"));
-  EXPECT_EQ(ellipse.get_name(), "A");
   EXPECT_FALSE(ellipse.get_center_state().is_empty());
```

Comment:
Setting the center state now also counts as non-empty

```suggestion
  EXPECT_FALSE(ellipse.is_empty());
  EXPECT_FALSE(ellipse.get_center_state().is_empty());
```